sql/dd/dd_table.cc

```diff
@@ -2107,12 +2107,17 @@ static bool fill_dd_table_from_create_info(
       encrypt_type.assign(create_info->encrypt_type.str,
                           create_info->encrypt_type.length);
     }
+    table_options->set("encrypt_type", encrypt_type);
```

Comment:
"MySQL now adds clause ENCRYPTION=Y/N to all tables." I thought this is done by upstream. Why are 'we' doing it here?

Reply:
This comes from upstream and it was merged in from upstream in this PR.